tests/unit/datepicker/methods.js

```diff
@@ -8,9 +8,18 @@ define( [
 QUnit.module( "datepicker: methods" );
 
 QUnit.test( "destroy", function( assert ) {
-	assert.expect( 33 );
+	assert.expect( 35 );
 	var inl,
-		inp = testHelper.init( "#inp" );
+		inp = testHelper.init( "#inp" ),
+		dp = $( "#ui-datepicker-div" );
+
+	// Destroy and clear active reference
+	inp.datepicker( "show" );
+	assert.equal( dp.css( "display" ), "block", "Datepicker - visible" );
+	inp.datepicker( "hide" ).datepicker( "destroy" );
+	assert.ok( !$.datepicker._curInst, "Datepicker - destroyed and cleared reference" );
```

Comment:
Could you make it more specific here?
```suggestion
	assert.ok( $.datepicker._curInst == null, "Datepicker - destroyed and cleared reference" );
```

Reply:
@mgol that make sense. Thanks !!